performant-software/Annotation-Studio
# Confirm annotation deletion

## What does this PR do?
1. Before users delete an annotation message they receive a confirmation message

## How to test?
1. Login and go to http://cove-staging.herokuapp.com/documents/sartor-resartus-three with your admin account
2. Switch the user to Jamie using the Switch User dropdown 
3. Now use the By User dropdown and switch to Jamie. 
4. Hover on the annotation and click on the x in the upper right hand corner of the annotation edit popup.
5. You should receive a confirmation message. Click on cancel and confirm the annotation is still there.
6. Click on the x again and make sure the annotation is deleted.
